include/mbedtls/config.h

```diff
@@ -2728,6 +2728,27 @@
  */
 #define MBEDTLS_ERROR_C
 
+/**
+ * \def MBEDTLS_CRC_C
+ *
+ * Enable the CRC calculating module
+ *
+ * Module:  library/crc.c
+ *
+ * This module enables mbedtls_crc_update.
+ */
+//#define MBEDTLS_CRC_C
+
+/**
+ * \def MBEDTLS_VALIDATE_AES_KEYS
+ *
+ * Enable validation of AES keys by checking their CRC
+ * after every encryption/decryption.
```

Comment:
It should be "during" enc/dec, not after

Reply:
According to the security doc, a failure, or identifying a wrong key, should not affect the function flow, as it might help the attacker to find the vulnerable spot in runtime.

Reply:
@AndrzejKurek , I mean for flag description it should be "during" or "per" enc/dec, because for user it is located in enc/dec function and not after it

Reply:
Oh, sure, I can change that.

Reply:
Done.